### PR TITLE
Added example of using Firebase to docs

### PR DESCRIPTION
**Pull request by limitlessloop**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI/CSS related code, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

Shows an example of how to deploy a Saber project using Firebase hosting

- [ ] Briefly describing what this PR does in the title in [Angular Commit Message Conventions](https://git.io/fNGDG), e.g. `fix: rebuild when a page is added`

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

Feel free to tweak it to suit any naming conversions or best practices you think are appropriate.

### CODE REVIEW

File: website/pages/docs/deployment.md [hunk 1]
```diff
@@ -125,6 +125,77 @@ Then run the command `now` in your project, you will be given a `now.sh` URL as
...
 
+## [Firebase](https://firebase.google.com/)
+
+To host and deploy your site to Firebase.
+
+### Step 1: Log into Firebase and create a new project or choose from an existing one
+
+### Step 2: Create a new web app from the project dashboard and select "Also setup Firebase Hosting for this app"
+
+### Step 3: Add the required Firebase scripts to the body of your site by editing a file called `saber-browswer.json` at the root of your project to include the following.
```
Comment (egoist) [inline website/pages/docs/deployment.md]: this is probably unnecessary.

Comment (limitlessloop) [reply website/pages/docs/deployment.md]: Sure, whatever you feel is appropriate. Depends how much hand-holding you want to give.

File: website/pages/docs/deployment.md [hunk 2]
```diff
@@ -125,6 +125,76 @@ Then run the command `now` in your project, you will be given a `now.sh` URL as
...
+## [Firebase](https://firebase.google.com/)
+
+To host and deploy your site to Firebase.
+
+### Step 1: Create a new web app and select "Also setup Firebase Hosting for this app"
+
+### Step 2: Add the required Firebase scripts to the body of your site
+
+```js
+// sabe.browser.js
```
Comment (MarchWorks) [inline website/pages/docs/deployment.md]: Typo

Comment (limitlessloop) [reply website/pages/docs/deployment.md]: Thank you

File: website/pages/docs/deployment.md [hunk 3]
```diff
@@ -125,6 +125,76 @@ Then run the command `now` in your project, you will be given a `now.sh` URL as
 
 Check out [Now for GitHub](https://zeit.co/docs/v2/integrations/now-for-github) and [Now for GitLab](https://zeit.co/docs/v2/integrations/now-for-gitlab/) for continuous intergration.
 
+## [Firebase](https://firebase.google.com/)
+
+To host and deploy your site to Firebase.
+
+### Step 1: Create a new web app and select "Also setup Firebase Hosting for this app"
+
+### Step 2: Add the required Firebase scripts to the body of your site
```
Comment (egoist) [inline website/pages/docs/deployment.md]: Maybe just remove the step, you don't really the Firebase scripts for a static site.

Comment (limitlessloop) [reply website/pages/docs/deployment.md]: Firebase provides analytics with those scripts, if you remove the step then there isn't really much point in having any Firebase instructions.

**Pull request merged**